lib/matplotlib/ticker.py

```diff
@@ -689,20 +707,17 @@ def _set_offset(self, range):
             if np.absolute(ave_oom - range_oom) >= 3:  # four sig-figs
                 p10 = 10 ** range_oom
                 if ave_loc < 0:
-                    self.set_useOffset((math.ceil(np.max(locs) / p10) * p10))
+                    return math.ceil(np.max(locs) / p10) * p10
                 else:
-                    self.set_useOffset((math.floor(np.min(locs) / p10) * p10))
+                    return math.floor(np.min(locs) / p10) * p10
             else:
-                self.set_useOffset(False)
-
```

Comment:
Due to grouping methods (user, inherited, local), the diff is somewhat hard to read.